modules/navier_stokes/src/fvkernels/INSFVTKEDSourceSink.C

```diff
@@ -148,7 +155,7 @@ INSFVTKEDSourceSink::computeQpResidual()
         destruction += 2.0 * TKE * wall_mut / Utility::pow<2>(distance_vec[i]) / tot_weight;
       }
       else
-        destruction += std::pow(_C_mu, 0.75) * rho * std::pow(TKE, 1.5) /
+        destruction += std::pow(_C_mu, 0.75) * rho * std::pow(std::max(ADReal(0), TKE), 1.5) /
```

Comment:
TKE should be always positive by the conservation properties of the k epsilon mode. This capping may be unnecessary